src/installer/corehost/cli/bundle/extractor.cpp

```diff
@@ -33,16 +33,22 @@ pal::string_t& extractor_t::extraction_dir()
         pal::string_t host_name = strip_executable_ext(get_filename(m_bundle_path));
         if (!pal::is_path_rooted(m_extraction_dir))
         {
-            pal::string_t current_dir = _X(".");
+            pal::string_t cwd;
             pal::string_t relative_path(m_extraction_dir);
-            if (!pal::realpath(&current_dir))
+            if (_X('/') != DIR_SEPARATOR)
+            {
+                replace_char(&relative_path, _X('/'), DIR_SEPARATOR);
+            }
```

Comment:
I don't think we should do this. If nothing else it's not symmetrical: `my/path` will work on Windows, but `my\path` won't work on Linux. I assume you copied this from `deps_entry.cpp` - there it has a reason - in `.deps.json` we store all paths in the unix notation since we need that exact same file to work on all platforms. I don't think we should apply the same logic to environment variables.

Also Windows tends to accept `/` as directory separator in some cases, so it might work for the typical cases anyway.